clusterloader2/api/types.go

```diff
@@ -116,10 +116,12 @@ type NamespaceConfig struct {
 	// NamePrefix is the name prefix of automanaged namespaces.
 	// It's optional, if set CL will use it, otherwise generate one with random string.
 	Prefix string `json: prefix,omitempty`
-	// DeleteStaleNamespaces specifies whether or not delete stale namepaces
+	// DeleteStaleNamespaces specifies whether or not delete stale namepaces.
```

Comment:
nit: actually can you fix all of these to "namespaces"

Reply:
@coltonstapper fixed. thanks!